hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/WorkflowsInjectionModule.java

```diff
@@ -32,4 +36,9 @@
             PreHandleWorkflowInjectionModule.class,
             QueryWorkflowInjectionModule.class
         })
-public interface WorkflowsInjectionModule {}
+public interface WorkflowsInjectionModule {
+
+    @Binds
+    @Singleton
+    HederaRecordCache bindHederaRecordCache(RecordCacheImpl recordCache);
+}
```

Comment:
Makse sense!
I added the `HederaRecordCache` mapping together with the `RecordCache` because otherwise the classes that require `RecrodCache` are complaining.

An interesting thing is that under the hood Dagger uses the same recordCacheImplProvider for both methods but if I remove the HederaRecordCache binding it complains. I guess this is how Dagger works 🤷 
```
@Override
public RecordCache recordCache() {
  return recordCacheImplProvider.get();
}

@Override
public HederaRecordCache hederaRecordCache() {
  return recordCacheImplProvider.get();
}
```